misc/scripts/install-local.sh

```diff
@@ -178,7 +178,14 @@ function makeVirtualDeb {
 		fi
 
 		printf "Suggests:" |sudo tee -a "$SRCDIR/$name-pacstall/DEBIAN/control" > /dev/null
-		printf " %s\n" "${optdepends[@]}" | awk -F': ' '{print $1}' | tr '\n' ',' | head -c -2 | sudo tee -a "$SRCDIR/$name-pacstall/DEBIAN/control" > /dev/null
+		for i in "${optdepends[@]}"; do
+			if ! grep -q ':' <<< "${i}"; then
+				fancy_message error "${i} does not have a description"
```

Comment:
Should we error out or just add warning and rework the splitting method?

Reply:
Well, `optdepends` is supposed to be supplemental packages, so it might not immediately make sense why a package is being suggested, without a description.